airbase/pom.xml

```diff
@@ -450,6 +452,7 @@
                             <java.awt.headless>true</java.awt.headless>
                             <java.util.logging.SimpleFormatter.format>%1$tY-%1$tm-%1$tdT%1$tH:%1$tM:%1$tS.%1$tL%1$tz %4$s %5$s%6$s%n</java.util.logging.SimpleFormatter.format>
                         </systemPropertyVariables>
+                        <skipTests>${air.test.skip}</skipTests>
```

Comment:
When `air.test.skip` is true, we could perhaps exclude the whole `maven-surefire-plugin`, could we?
re-sharing https://peter.palaga.org/presentations/211021-baselone-maven-my-life-is-short/ shared with me by  @nineinchnick